clang/test/SemaCXX/warn-unsafe-buffer-usage-no-libc-functions-in-c.c

```diff
@@ -0,0 +1,9 @@
+// RUN: %clang_cc1 -Wunsafe-buffer-usage %s -verify %s
```

Comment:
FWIW the language mode is set via `-x` command line option. It is only inferred from the file extension if it's missing.